docs/section21/Rule21-1.md

```diff
@@ -0,0 +1,47 @@
+
+# Boiler - Rule 21-1  
+
+**Rule ID:** 21-1  
+**Rule Description:** For systems using purchased hot water or steam, the heating source shall be modeled as
+purchased hot water or steam in both the proposed design and baseline building design. If any system in the proposed design uses purchased hot water or steam, all baseline systems with hot water coils shall use the same type of purchased hot water or steam.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.1.1 & G3.1.1.3.1 Building System-Specific Modeling Requirements for the Baseline model  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-1, 5, 7, 11, or 12.
+
```

Comment:
1. @mkarpman I am unsure about what this RDS is doing. I've missed some calls so maybe this was discussed. If P-RMR is modeled with purchased hot water, then  the baseline is not necessarily system type 1,5,7,11 or 12 if the cooling is also purchased chilled water. In that case system 1 is replaced with FPFCU etc.
2. This rule isn't checking anything for the B-RMR, only the P-RMR. Is that not the intent of the RDS?

Reply:
@supriyagoel I agree with your first comment - we should remove this applicability check.
Aside from that, we agreed to limit implementation of this rule to just checking if it applies to the project, which is determined based on P-RMR. Notice that the rule doesn't have PASS or FAIL outcome. The only options are either (A) "UNDERTERMINED" (meaning that the manual review is required) + is_applicable=TRUE, or (B) no outcome except is_applicable=FLASE

Reply:
makes sense. @xingcx please remove this applicability check